### DISCUSSION TITLE
Can the host param be omitted within a ssh session?

### DISCUSSION THREAD

**Discussion by rayw0ng** [RsyncProject/rsync#421] [Q&A]
When login to host B from host A using ssh, we can use `rsync path hostA:path`. 

In the ssh session, normally we can tell the IP from $SSH_CLIENT, is it possible to omit `hostA` and use  `rsync path :path` instead?

**Comment by WayneD** [comment]
You'll need to use something like `rsync -ai src/ $(echo $SSH_CLIENT | awk '{print $1}'):dest/`

You could also put the sub-command into your .bashrc or similar to define a variable, making it easier to type a $VAR reference in a command: `export FROM=$(echo $SSH_CLIENT | awk '{print $1}')`

**Reply by rayw0ng** [reply]
Thanks for your answer. 

But I'm not asking how to use rsync, I'm asking is it feasible to add this **feature** to rsync.

**Comment by WayneD** [comment]
Sorry, that won't be added.
